src/gst-plugins-ugly.mk

```diff
@@ -25,4 +25,8 @@ define $(PKG)_BUILD
         --docdir='$(1)/sink' \
         --with-html-dir='$(1)/sink'
     $(MAKE) -C '$(1)' -j '$(JOBS)' install
+
+    # some .dlls are installed to lib - no obvious way to change
+    $(and $(BUILD_SHARED),
```

Comment:
Because I had just copied it from gstreamer.mk.
gstreamer.mk  doesn't use a indent for it.

Do you insist on change "and->if"?